docs/operational_guide/placement_configuration.md

```diff
@@ -126,16 +126,31 @@ After sending the add command you will need to wait for the M3DB cluster to reac
 
 #### Removing a Node
 
-Send a DELETE request to the `/api/v1/placement/<NODE_ID>` endpoint.
+Send a DELETE request to the `/api/v1/services/m3db/placement/<NODE_ID>` endpoint.
```

Comment:
nice thanks, I think the legacy stuff still works but good to update the docs to the new stuff